<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formLogin</class>
 <widget class="QDialog" name="formLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>338</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Inicio de sesion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(129, 61, 156);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../../Descargas/pull-shark-default.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(220, 138, 221);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>Inicio sesion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
   <property name="text">
    <string>Usuario:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>WaitCursor</cursorShape>
   </property>
   <property name="text">
    <string>Constraseña:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:solid 1px #ccccc;</string>
   </property>
   <property name="placeholderText">
    <string>Ingrese su uruario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Ingrese su contraseña</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcceder">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 162, 105);
border-radius: 8px</string>
   </property>
   <property name="text">
    <string>Acceder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string>Mensaje</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../../../../../Escritorio/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
